portality/templates/application_form/editorial_form_body.html

```diff
@@ -1,3 +1,10 @@
+{% if current_journal and not current_journal.is_in_doaj() %}
```

Comment:
This checks whether the current journal is not withdrawn. It is not defined if we open the journal form. How to check whether journal has been deleted? I guess current_journal will exist but will it be just empty string?